doc/content/verification_and_validation/ver-1kc.md

```diff
@@ -0,0 +1,83 @@
+# ver-1kc
+
+# Sieverts’ Law Boundaries with No Volumetric Source
+
+## General Case Description
+
+Two enclosures are separated by a membrane that allows diffusion according to Sievert's law, with no volumetric source present. Enclosure 2 has twice the volume of Enclosure 1.
+
+## Case Set Up
+
+This verification problem is taken from [!cite](ambrosek2008verification).
+
+This setup describes a diffusion system in which tritium T$_2$ is modeled across a one-dimensional domain split into two enclosures. The total system length is $2.5 \times 10^{-4}$ m, divided into 100 segments. The system operates at a constant temperature of 500 Kelvin. Initial tritium pressures are specified as $10^{5}$ Pa for Enclosure 1 and $10^{-10}$ Pa for Enclosure 2.
+
+The diffusion process in each of the two enclosures can be described by
+
+\begin{equation}
+\frac{\partial C_1}{\partial t} = \nabla D \nabla C_1,
+\end{equation}
+and
+\begin{equation}
+\frac{\partial C_2}{\partial t} = \nabla D \nabla C_2,
+\end{equation}
+
+where $C_1$ and $C_2$ represent the concentration fields in enclosures 1 and 2 respectively, $t$ is the time, and $D$ denotes the diffusivity.
+
+The concentration in Enclosure 1 is related to the partial pressure and concentration in Enclosure 2 via the interface sorption law:
+
+\begin{equation}
+C_1 = K P_2^n = K \left( C_2 RT \right)^n
+\end{equation}
+
+where $R$ is the ideal gas constant in J/mol/K, $T$ is the temperature in K, $K$ is the solubility, and $n$ is the exponent of the sorption law. For Sieverts' law, $n=0.5$.
+
+## Results
+
+Two subcases are considered. In the first subcase, we assume that $K=1/RT$, which is expected to lead to $C_1=C_2^{1/2}$ at equilibrium. In the second, $K=10/RT$, which is expected to lead to $C_1 = 10 C_2^{1/2}$. This second case is added to exercise TMAP8 in a case with a concentration jump.
```

Comment:
Instead of using K=1/RT and K=10/RT, you should use K=1/sqrt(RT) and K=10/sqrt(RT) to account for n=0.5. 

Again, K=1/sqrt(RT) would not lead to a concentration jump, but K=10/sqrt(RT) would.

Reply:
Forget this comment, C_1 = 1/sqrt(RT) sqrt(C_2 RT) = sqrt(C_2) != C_2. So there would still be a concentration jump. 

Just do K=10/sqrt(RT).